src/ml_tooling/transformers.py

```diff
@@ -54,23 +62,22 @@ def _most_freq(X):
                           'min': pd.DataFrame.min}
 
     def fit(self, X: pd.DataFrame, y=None):
-        if self.value is None and self.strategy is None:
```

Comment:
Maybe abstract this into a class method?
```
def validate_input(strategy, value):
    ** error handling**
   return strategy, value
```

then in `__init__`
```python
def __init__(*args, **kwargs):
    self.strategy, self.value = self.validate_input(strategy, value)
```

Reply:
Done :-)